src/backend/index.ts

```diff
@@ -2,16 +2,633 @@ import express from 'express';
 import * as functions from 'firebase-functions';
 import next from 'next';
 
-import { spotifyCallbackHandler } from './handler';
+import { addSpotifyPlaylistHandler, spotifyCallbackHandler } from './handler';
+import { Album, Playlist, SpotifyPlaylist, Track } from '../types';
+import { saveAlbum, savePlaylist, saveTrack } from './lib/firestore';
+import { fireStore } from './lib/firebase';
 
 const server = express();
 const nextjsServer = next({
   conf: { distDir: 'lib/next' },
 });
 const nextjsHandle = nextjsServer.getRequestHandler();
 
-server.get('/callback', spotifyCallbackHandler);
+server.get('/save', async (req, res, next) => {
```

Comment:
```suggestion
server.get('/save', async (req, res) => {
```

next使ってないじゃんってGitHub Actionsがいってる